spec/requests/v0/preneeds/burial_forms_spec.rb

```diff
@@ -16,7 +16,7 @@
 
     let(:submission_record) { OpenStruct.new(application_uuid: 'UUID') }
     let(:form) do
-      Preneeds::BurialForm.new(params).tap do |f|
+      Preneeds::BurialForm.new(params[:application]).tap do |f|
```

Comment:
You'd need to update other places that call `new` (e.g. [mobile preneeds controller](https://github.com/department-of-veterans-affairs/vets-api/blob/eb668ed6770a883f0cb1fb3484903c75a4db69e6/modules/mobile/app/controllers/mobile/v0/pre_need_burial_controller.rb#L11))

Reply:
`applicant` is incorrect wrapping the params in the spec. 

```
def burial_form_params
  params.require(:application).permit(..omit..)
end
```